client/js/components/GameView/MoveChoiceOverlay.vue

```diff
@@ -86,23 +86,33 @@ export default {
         cardRank = this.selectedCard.rank;
       }
       // Re-usable move-objects
+      const allMovesAreDisabled = !this.isPlayersTurn || this.frozenId === this.selectedCard.id;
+      const allMovesDisabledExplanation = !this.isPlayersTurn
+        ? "It's not your turn"
+        : 'This card is frozen';
+
+      let disabledText;
+      if (allMovesAreDisabled) {
+        disabledText = allMovesDisabledExplanation;
+      }
+
       // Points
       const pointsDescription = `Gain ${cardRank} point` + (cardRank === 1 ? '' : 's');
       const pointsMove = {
         displayName: 'Points',
         eventName: 'points',
         moveDescription: pointsDescription,
         disabled: !this.isPlayersTurn,
```

Comment:
Seems like you could do this
```suggestion
        disabled: !this.isPlayersTurn || allMovesAreDisabled,
```